spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -792,9 +793,9 @@ July 27, 2020 - Re-addition of source field
 
 Nov 11, 2022 - Addition of a memo field
 
-Sep 22, 2023 - Support for multi-token packets
+Sep 22, 2023 - Support for multi-token packets - [PR](https://github.com/cosmos/ibc/pull/1020)
 
-March 5, 2024 - Support for ics20-2
+March 5, 2024 - Support for ics20-2 - [PR](https://github.com/cosmos/ibc/pull/1090)
 
```

Comment:
Even though the [PR to use protobuf encoding](https://github.com/cosmos/ibc/pull/1118) was merged into the path forwarding branch, maybe it's worth to add an entry for it in the history section.

Reply:
Agreed and [applied](https://github.com/cosmos/ibc/pull/1124/commits/e5b9e733a34c6fc2df83bad63e7f1b7641fe9e84)!